content/nextflow-on-aws-batch/40_nextflow202/31_od.md

```diff
@@ -1,20 +1,19 @@
 ---
-title: "On-Demand"
+title: "OD Compute Environment"
 chapter: false
 weight: 31
 ---
 
 ## Create EC2 On-Demand Compute Environment
 
-The nextflow tasks, which are supervising the orchestration of the actual workflow are not able to tolerate interruptions as they need to run for the duration of the complete workflow.
-Thus, we create a small On-Demand compute environment which will run the supervision task.
+Similar to what we did for the **spot-ce** used by the **job-queue** we are going to create a Compute Environment which uses a couple of large instances to run the Nextflow process within a AWS Batch job.
 
 ### Setup Compute ENvironment
```

Comment:
Environment.

_I know it was not changed in this PR, but I like to live in danger_

Reply:
done